### PR TITLE
docs(README): add list adapters section

### LINKED ISSUE #3218: Section to explain how to generate a list of available adapters in the README
I wanted to make a small yet quite handy (imo) contribution to the `README.md` by simply adding a section to explain how to generate a list of all the available adapters; as this feature has already been implemented, still, it may not be straightforward for the generic user (unless they look into the `package.json`)

### PR DESCRIPTION
**Pull request by czar0**
## Description

A section in the README to explain how to generate a list of all available adapters.

## Changes

A bunch of lines in the README (only docs) and one line in .gitignore.

## Steps to Test

N/A

## Quality Assurance

N/A

### CODE REVIEW

**Comment by karen-stepanyan** [comment]
Hey @czar0 . Thanks for opening the PR. What's the intention of this change ? The `MASTERLIST.md` is created/updated automatically after every release, so it contains up-to-date information and is available at a root level. Why would anyone want to generate it manually ?

**Comment by czar0** [comment]
oh, shoot! I wondered whether that script was even called anywhere😄 (apologies). Anyway, this unclarity builds even a stronger case for this simple addition. The intent is to slightly improve the user experience by presenting to the end user a list of available adapters (the culprit `MASTERLIST.md`).
Put it in a story format: _As an end user, I want to have a concise list of available external adapters and their properties so that I can quickly understand their purpose and start using them_.
I believe a small addition to the README should be enough (changes just pushed).
Let me know if that makes sense to you.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ Clears all build files/directories. Useful in case of issues when installing dep
     ╚══ targets - adapters that write data to a location, often a blockchain.
 ```
 
+## Adapters
+
+The [MASTERLIST.md](./MASTERLIST.md) file in the root directory contains the list of all the external adapters available in this repository, together with their version, type, level of support, and other helpful information. Check their linked README to learn more about them.
```
Comment (karen-stepanyan) [inline README.md:77]: Do you mind to change this to 
```suggestion
The [MASTERLIST.md](./MASTERLIST.md) file in the root directory contains the list of all the external adapters available in this repository, together with their version, type, supported environment variables, endpoints and other helpful information. Check their linked README to learn more about them.
```

**Review by karen-stepanyan — CHANGES_REQUESTED** [review]
I agree that we can include information about Masterlist in the main readme. Suggested a minot change

**Review by karen-stepanyan — APPROVED** [review]

**Pull request merged**